### PR TITLE
Init Model

### PR DESCRIPTION
**Pull request by Linh-PCY**
==> Title: Init Model

==> Redmine: https://edu-redmine.sun-asterisk.vn/projects/oe24-clothes-rover-shop/issues

==> Attachment File:
Screenshot from 2019-11-15 12-05-07

### CODE REVIEW

**Comment by Linh-PCY** [comment]
Ready

File: test/fixtures/blogs.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (phucmh-1356) [inline test/fixtures/blogs.yml]: thêm test/* vào gitignore đi em

**Comment by Linh-PCY** [comment]
ready

**Comment by Linh-PCY** [comment]
ready

**Comment by manhbnt-0960** [comment]
commented
cho folder test vào git ignore đi e

**Comment by Linh-PCY** [comment]
ready

**Pull request merged**